cmd/dcrdata/views/blocks.tmpl

```diff
@@ -18,7 +18,7 @@
             <div class="d-flex justify-content-between align-items-end">
                 <span class="h2 d-flex pt-4 pb-1 pe-2">
                     Blocks
-                    <span class="dcricon-info fs24 ms-3 mt-2"></span>
+                    <span class="dcricon-info fs24 ms-3 mt-2" title="List of Decred Blocks"></span>
```

Comment:
As I mentioned in https://github.com/decred/dcrdata/issues/1939#issuecomment-1338022812, I still think the info icon for "Blocks" is silly.  It's obviously a list of Decred blocks.

Reply:
> I agree but then it's design stuff, are you suggesting we remove it? IMO keeping it makes the design uniform.

Yeah. I don't think there needs to be an info icon on everything.  I'm pretty sure these icons with no `title` attributes were half-finished work.  If something needs more detailing, it can have the icon.

Also, the icon is really huge.  Normally these things are quite small, even smaller than the text they accompany, with a top valign.  How would that look?